core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorCapabilities.java

```diff
@@ -17,4 +17,5 @@
 {
     NOT_NULL_COLUMN_CONSTRAINT,
     MATERIALIZED_VIEW_GRACE_PERIOD,
+    DROP_SCHEMA_CASCADE,
```

Comment:
i don't thnk we need a new capability, since the functionality is implemented with a new ConnectorMetadata method.